index.html

```diff
@@ -1,19 +1,138 @@
+                <li>
+                    <div class="mb-3">
+                        <label for="text" class="form-label">First Commit Hash</label><input type="text" id="firstCommit" class="form-control">
+                        <label for="text" class="form-label">Second Commit Hash</label><input type="text" id="secondCommit" class="form-control">
+                    </div>
+                    <center><button class="btn btn-block btn-primary" type="submit" id="commitsSubmit">Load commit diff</button></center>
+                </li>
+                <li class="border-top my-3"></li>
+                <li class="mb-1">
+                    <button class="btn btn-toggle align-items-center rounded collapsed" data-bs-toggle="collapse" data-bs-target="#dates-collapse" aria-expanded="false">
+                        Date Presets
+                    </button>
+                    <div class="collapse" id="dates-collapse">
+                        <ul class="btn-toggle-nav list-unstyled fw-normal pb-1 small" id="datesPresets">
+                        </ul>
+                    </div>
+                </li>
+                <li class="mb-1">
+                    <button class="btn btn-toggle align-items-center rounded collapsed" data-bs-toggle="collapse" data-bs-target="#flavors-collapse" aria-expanded="false">
+                        Flavor Presets
+                    </button>
+                    <div class="collapse" id="flavors-collapse">
+                        <ul class="btn-toggle-nav list-unstyled fw-normal pb-1 small" id="flavorsPresets">
+                        </ul>
+                    </div>
+                </li>
+                <li class="mb-1">
+                    <button class="btn btn-toggle align-items-center rounded collapsed" data-bs-toggle="collapse" data-bs-target="#charts-collapse" aria-expanded="false">
+                        Charts Presets
+                    </button>
+                    <div class="collapse" id="charts-collapse">
+                        <ul class="btn-toggle-nav list-unstyled fw-normal pb-1 small" id="chartsPresets">
+                        </ul>
+                    </div>
+                </li>
+                <li class="mb-1">
+                    <button class="btn btn-toggle align-items-center rounded collapsed" data-bs-toggle="collapse" data-bs-target="#legend-collapse" aria-expanded="true">
+                        Chart Legend
+                    </button>
+                    <div class="collapse show" id="legend-collapse">
+                        <ul class="btn-toggle-nav list-unstyled fw-normal pb-1 small" id="chartLegend">
+                        </ul>
+                    </div>
+                </li>
+            </ul>
+        </div>
+        <div class="b-example-divider" style="height: auto !important"></div>
+        <div style="width:85%; overflow-x: hidden; overflow-y: auto; text-align: justify; max-height: auto !important" id="charts" class="p-3">
```

Comment:
I wouldn't disable horizontal scrolling, because if your browser is too small, you won't right sides of the graphs.

```suggestion
        <div style="width:85%; overflow-x: auto; overflow-y: auto; text-align: justify; max-height: auto !important" id="charts" class="p-3">
```